app/models/owner.js

```diff
@@ -67,6 +69,36 @@ export default VcsEntity.extend({
     }, { live: false });
   },
 
+  fetchPlans: task(function* () {
+    const url = this.isOrganization ? `/plans_for/organization/${this.id}` : '/plans_for/user';
+    const result = yield this.api.get(url);
+    return result ? result.plans : [];
+  }).keepLatest(),
+
+  fetchPlansInstance: computed(function () {
+    return this.fetchPlans.perform();
+  }),
+
+  isFetchPlansRunning: reads('fetchPlansInstance.isRunning'),
+  eligiblePlans: reads('fetchPlansInstance.value'),
```

Comment:
I'd suggest these properties to derive from `fetchPlans` task. The `fetchPlans.perform` returns a promise, which then resolves to a value.
```suggestion
  isFetchPlansRunning: reads('fetchPlans.isRunning'),
  eligiblePlans: reads('fetchPlans.lastSuccessful.value'),
```

Reply:
Hey @emmanuel-ci, thanks for the details, it totally makes sense to me!